### PR TITLE
Make nsxt managed objects findable in opensearch

### PR DESCRIPTION
**Pull request by sven-rosenzweig**
Searching for objects based on their UUID in Opensearch does not return the corresponding log line of type `Resource:<resource> with ID:<uuid> is going to be <action>.`. Opensearch seems to run word searches rather than substring searches, therefor searching for the UUID will not return the information needed.

### CODE REVIEW

**Review by joker-at-work — COMMENTED** [review]
I think you missed some places:
```
$ git grep 'ID:%s'
networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/cli.py:                        LOG.info("Object with ID:%s already processed.", id)
networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_mgmt.py:                "Resource:%s with ID:%s deletion is rescheduled due to dependency.", Provider.SG_MEMBERS, sg.get("id")
networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_policy.py:    RESCHEDULE_WARN_MSG = "Resource:%s with ID:%s deletion is rescheduled due to dependency."
networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_policy.py:                LOG.info("%s ID:%s in Status:%s", resource_type, os_id, status)
networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_policy.py:                LOG.info("%s ID:%s in Status:%s for %ss", resource_type, os_id, status, attempt * pause)
networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_policy.py:            LOG.info("Resource:%s with ID:%s already deleted.", resource_type, os_id)
$ git grep 'ID:{'
networking_nsxv3/common/synchronization.py:MESSAGE = "{} Resource ID:{} with Priority:{} for action {}"
networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_mgmt.py:        begin_report = "[{}] Resource:{} with ID:{} is going to be %s.".format(self.provider, resource_type, os_id)
networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_mgmt.py:        end_report = "[{}] Resource:{} with ID:{} has been %s.".format(self.provider, resource_type, os_id)
networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_policy.py:        raise Exception("{} ID:{} did not get realized for {}s", resource_type, os_id, until * pause)
networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_policy.py:        report = "Resource:{} with ID:{} is going to be %s.".format(resource_type, os_id)
networking_nsxv3/tests/unit/provider.py:                        return self.resp(417, "SG with ID:{} cannot be deleted as either it has children or it is being referenced.".format(real_id))
```

EDIT: Forgot to mention, that I really like this PR. Thank you :)

**Comment by sven-rosenzweig** [comment]
> I think you missed some places:
> 
> ```
> $ git grep 'ID:%s'
> networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/cli.py:                        LOG.info("Object with ID:%s already processed.", id)
> networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_mgmt.py:                "Resource:%s with ID:%s deletion is rescheduled due to dependency.", Provider.SG_MEMBERS, sg.get("id")
> networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_policy.py:    RESCHEDULE_WARN_MSG = "Resource:%s with ID:%s deletion is rescheduled due to dependency."
> networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_policy.py:                LOG.info("%s ID:%s in Status:%s", resource_type, os_id, status)
> networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_policy.py:                LOG.info("%s ID:%s in Status:%s for %ss", resource_type, os_id, status, attempt * pause)
> networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_policy.py:            LOG.info("Resource:%s with ID:%s already deleted.", resource_type, os_id)
> $ git grep 'ID:{'
> networking_nsxv3/common/synchronization.py:MESSAGE = "{} Resource ID:{} with Priority:{} for action {}"
> networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_mgmt.py:        begin_report = "[{}] Resource:{} with ID:{} is going to be %s.".format(self.provider, resource_type, os_id)
> networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_mgmt.py:        end_report = "[{}] Resource:{} with ID:{} has been %s.".format(self.provider, resource_type, os_id)
> networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_policy.py:        raise Exception("{} ID:{} did not get realized for {}s", resource_type, os_id, until * pause)
> networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_policy.py:        report = "Resource:{} with ID:{} is going to be %s.".format(resource_type, os_id)
> networking_nsxv3/tests/unit/provider.py:                        return self.resp(417, "SG with ID:{} cannot be deleted as either it has children or it is being referenced.".format(real_id))
> ```
> 
> EDIT: Forgot to mention, that I really like this PR. Thank you :)



> I think you missed some places:
> 
> ```
> $ git grep 'ID:%s'
> networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/cli.py:                        LOG.info("Object with ID:%s already processed.", id)
> networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_mgmt.py:                "Resource:%s with ID:%s deletion is rescheduled due to dependency.", Provider.SG_MEMBERS, sg.get("id")
> networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_policy.py:    RESCHEDULE_WARN_MSG = "Resource:%s with ID:%s deletion is rescheduled due to dependency."
> networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_policy.py:                LOG.info("%s ID:%s in Status:%s", resource_type, os_id, status)
> networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_policy.py:                LOG.info("%s ID:%s in Status:%s for %ss", resource_type, os_id, status, attempt * pause)
> networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_policy.py:            LOG.info("Resource:%s with ID:%s already deleted.", resource_type, os_id)
> $ git grep 'ID:{'
> networking_nsxv3/common/synchronization.py:MESSAGE = "{} Resource ID:{} with Priority:{} for action {}"
> networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_mgmt.py:        begin_report = "[{}] Resource:{} with ID:{} is going to be %s.".format(self.provider, resource_type, os_id)
> networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_mgmt.py:        end_report = "[{}] Resource:{} with ID:{} has been %s.".format(self.provider, resource_type, os_id)
> networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_policy.py:        raise Exception("{} ID:{} did not get realized for {}s", resource_type, os_id, until * pause)
> networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_policy.py:        report = "Resource:{} with ID:{} is going to be %s.".format(resource_type, os_id)
> networking_nsxv3/tests/unit/provider.py:                        return self.resp(417, "SG with ID:{} cannot be deleted as either it has children or it is being referenced.".format(real_id))
> ```
> 
> EDIT: Forgot to mention, that I really like this PR. Thank you :)

True, thanks for pointing this out. Added spaces to the missing log lines

File: networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_mgmt.py [hunk 1]
```diff
@@ -738,7 +738,7 @@ def qos_realize(self, qos, delete=False):
     def sg_members_realize(self, sg, delete=False):
         if delete and self.metadata(Provider.SG_RULES, sg.get("id")):
             LOG.warning(
-                "Resource:%s with ID:%s deletion is rescheduled due to dependency.", Provider.SG_MEMBERS, sg.get("id")
+                "Resource:%s with ID: %s deletion is rescheduled due to dependency.", Provider.SG_MEMBERS, sg.get("id")
```
Comment (joker-at-work) [inline networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_mgmt.py:741]: You didn't add the space after `Resource:` here.

Comment (sven-rosenzweig) [reply networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_mgmt.py]: Added

**Review by joker-at-work — APPROVED** [review]

**Pull request merged**